pipeline/projects/knoedler/__init__.py

```diff
@@ -723,10 +723,7 @@ def modifiers(self, a:dict, key:str):
 	def _empty_tx(self, data, incoming=False, purpose=None):
 		tx_uri = self.helper.transaction_uri_for_record(data, incoming)
 		tx_type = data.get('book_record', {}).get('transaction', 'Sold')
-		if purpose == 'returning':
-			tx = vocab.make_multitype_obj(vocab.SaleAsReturn, vocab.ProvenanceEntry, ident=tx_uri)
-		else:
-			tx = vocab.ProvenanceEntry(ident=tx_uri)
```

Comment:
This change seems outside the scope of the the description of AR-132. Can you please add context for why this is being done (was this based on a private conversation?), or revert it?